src/lib.rs

```diff
@@ -4135,12 +4139,12 @@ impl Build {
     /// search for |prog| on 'programs' path in '|cc| -print-search-dirs' output
     fn search_programs(
         &self,
-        cc: &mut Command,
+        clang: &Path,
```

Comment:
As mentioned elsewhere, I advocate in favour of using this method even with gcc. In other words I'd keep `cc` as variable name.